cmd/controller/disk.go

```diff
@@ -28,12 +28,16 @@ type DiskInfo struct {
 	Serial           string          // Serial is serial no of a disk
 	Vendor           string          // Vendor is vendor of a disk
 	Path             string          // Path is dev path of a disk like /dev/sda
+	FirmwareRevision string          // FirmwareRevision is the firmare revision for a disk
+	LogicalSize      uint32          // LogicalSize is the Logical size of a disk
+	Version          string          // Version is the disk version
```

Comment:
Is this the version of the firmware? I think so?

Reply:
My bad @gila .
The VERSION field indicates the implemented version of the standard such as SPC-1 or SPC-2, etc.